src/lib.rs

```diff
@@ -85,6 +85,7 @@ macro_rules! directory_entry(
 
 directory_entry!(ExportTable         = RVA<ExportDirectory>);
 directory_entry!(BaseRelocationTable = RVA<RelocationBlock>);
+directory_entry!(Debug               = RVA<DebugDirectory>);
```

Comment:
Could you clarify on `RVA<[DebugDirectory]>`, `[DebugDirectory]` is not RefSafe so wouldn't be able to implement the Directory trait.